libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -114,7 +114,6 @@ private const val MAPBOX_NAVIGATION_TOKEN_EXCEPTION = "A valid token is required
  * todo should we expose a "primaryRouteIndex" field instead of relying on the list's order?
  *
  * @param context activity/fragment's context
- * @param accessToken [Mapbox Access Token](https://docs.mapbox.com/help/glossary/access-token/)
```

Comment:
Could you add an entry somewhere in this top-level doc about the `defaultNavigationOptions` method?